src/main/java/uk/gov/hmcts/darts/audio/config/AudioConfiguration.java

```diff
@@ -1,10 +1,22 @@
 package uk.gov.hmcts.darts.audio.config;
 
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import java.time.Clock;
+
 @Configuration
 @EnableConfigurationProperties(AudioConfigurationProperties.class)
 public class AudioConfiguration {
 
+    @Value("${darts.audio.outbounddeleter.last-accessed-deletion-day:2}")
+    private long lastAccessedDeletionDays;
```

Comment:
Should this be used by something?

Reply:
its being used in the OutboundAudioDeleterProcessorImpl

Reply:
It is, but its being injected directly into the OutboundAudioDeleterProcessorImpl's constructor.  I'm fairly confident that if you were to remove the line above it would still be available in the OutboundAudioDeleterProcessorImpl.

Reply:
ACCEPT.